.github/workflows/generate_documentation.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           path: ~/SpacemanDMM
           key: ${{ runner.os }}-spacemandmm-${{ secrets.CACHE_PURGE_KEY }}
       - name: Install SpacemanDMM
-        run: bash tools/ci/install_spaceman_dmm.sh dmdoc
+        run: bash tools/ci/build_spaceman_dmm.sh dmdoc
```

Comment:
Ideally, we would always call `install_spaceman_dmm` first and then, if that fails, run `build_spaceman_dmm`.